biolink-model.yaml

```diff
@@ -883,6 +886,14 @@ slots:
     exact_mappings:
       - WIKIDATA_PROPERTY:P274
 
+  is_metabolite:
```

Comment:
@vdancik This should be `is metabolite` (without the underscore).

Reply:
Also, is this slot supposed to hold a boolean value? i.e. `is_metabolite: true`?

If so, then the range should be `boolean` and the domain should be `chemical substance`